test/functional/test_framework/test_framework.py

```diff
@@ -469,6 +470,7 @@ def _initialize_chain(self):
                     bitcoin_cli=self.options.bitcoincli,
                     mocktime=self.mocktime,
                     coverage_dir=None,
+                    cwd=self.options.tmpdir
```

Comment:
style-nit: Please add a trailing comma, otherwise formatters such as https://github.com/MarcoFalke/yapf-diff will grind on this